src/mn_snapshot.c

```diff
@@ -97,15 +105,31 @@ void MN_ReadSavegameTime(int i, char *name)
     }
     else
     {
+#if defined(_WIN32)
+        SYSTEMTIME lt;
+        wchar_t wdate[64];
+
+        GetLocalTime(&lt);
```

Comment:
Does this print the savegame file date?

Reply:
I am not sure what I was thinking, fixed here: https://github.com/fabiangreffrath/woof/pull/1631/commits/5f358670b454fa4e7675e95ae3d3a3d298ed5e32